xdsl/dialects/arith.py

```diff
@@ -780,6 +786,9 @@ def print(self, printer: Printer):
         printer.print_operand(self.lhs)
         printer.print(", ")
         printer.print_operand(self.rhs)
+        if self.fastmath is not None and self.fastmath != FastMathFlagsAttr("none"):
```

Comment:
I'm unsure that this printing is correct if there are other attributes on the function (is the attribute dictionary being printed anywhere?

Reply:
I don't think so, as nothing `fastmath`-related is printed when it's set as an attribute without this change